sourcemash/api/feeds.py

```diff
@@ -74,6 +79,14 @@ def post(self):
         if rss_feed['bozo'] == 1:
             return {"errors": {"url": ["URL is not a valid feed"]}}, 422
 
+        with open(BAD_WORDS_FILE) as data_file:
+            data = json.load(data_file)
+            badwords = data['badwords']
+            words = rss_feed['feed']['title'] + rss_feed['feed']['description']
```

Comment:
Could this lead to a weird middle world?

E.g. "I want to dance." + "Hello, World!" --> "I want to dance.Hello, World!"